service/internal/config/config.go

```diff
@@ -19,12 +20,13 @@ import (
 )
 
 type Config struct {
-	DevMode  bool                                     `mapstructure:"dev_mode"`
-	DB       db.Config                                `yaml:"db"`
-	OPA      opa.Config                               `yaml:"opa"`
-	Server   server.Config                            `yaml:"server"`
-	Logger   logger.Config                            `yaml:"logger"`
-	Services map[string]serviceregistry.ServiceConfig `yaml:"services" default:"{\"policy\": {\"enabled\": true}, \"health\": {\"enabled\": true}, \"authorization\": {\"enabled\": true}, \"wellknown\": {\"enabled\": true}, \"kas\": {\"enabled\": true}, \"entityresolution\": {\"enabled\": true}}"`
+	DevMode       bool                                     `mapstructure:"dev_mode"`
+	DB            db.Config                                `yaml:"db"`
+	OPA           opa.Config                               `yaml:"opa"`
+	Server        server.Config                            `yaml:"server"`
+	Logger        logger.Config                            `yaml:"logger"`
+	SensitiveKeys map[string]bool                          `yaml:"sensitiveKeys"`
```

Comment:
Can you make this a list instead of a dictionary?

```suggestion
	SensitiveKeys []string                          `yaml:"sensitiveKeys"`
```

Reply:
done